edge-detect/docker-compose.yml

```diff
@@ -3,13 +3,13 @@ version: '3.7'
 services:
```

Comment:
Is this file needed?

Reply:
Removed from https://github.com/IQTLabs/edgetech-coral-detect/pull/1